integration_tests/androidx_test/src/test/java/org/robolectric/integrationtests/axt/EspressoTest.java

```diff
@@ -115,6 +116,13 @@ public void textView() {
               assertThat(view.getHeight()).isGreaterThan(0);
             });
     onView(withText("Text View")).check(matches(isCompletelyDisplayed()));
+    onView(withText("Text View"))
```

Comment:
Maybe we can extract `Text View` to a string for this class.

Reply:
Yes, I thought about it, but I decided to follow the style in the whole class and do it consistently. But if you think it's better that way, no problem.

Reply:
Not a big deal. We can keep it for consistent.